docs/API/input.md

```diff
@@ -0,0 +1,29 @@
+![Input](https://i.imgur.com/XPecaeO.png)
+
+```js
+
+import { Input } from 'react-native-elements'
+
+<Input />
+
+
+
+```
+
+#### Input props
+
+| prop | default | type | description |
+| ---- | ---- | ----| ---- |
+| name | none | string | name of icon (required) |
+| type | material | string | type (defaults to material, options are `material-community, zocial, font-awesome, octicon, ionicon, foundation, evilicon, simple-line-icon, feather or entypo`) |
+| size | 26 | number | size of icon (optional) |
+| color | black | string | color of icon (optional) |
+| iconStyle | inherited style | object (style) | additional styling to icon (optional) |
+| component | View if no onPress method is defined, TouchableHighlight if onPress method is defined | React Native component | update React Native Component (optional) |
+| onPress | none | function | onPress method for button (optional) |
+| onLongPress | none | function | onLongPress method for button (optional) |
+| underlayColor | icon color | string | underlayColor for press event |
+| reverse | false | boolean | reverses color scheme (optional) |
+| raised | false | boolean | adds box shadow to button (optional) |
+| containerStyle | inherited styling | object (style) | add styling to container holding icon (optional) |
+| reverseColor | white | string | specify reverse icon color (optional) |
```

Comment:
@deadcoder0904 can we do this in another PR? That way we don't add mix up multiple things in 1 PR. 

So please delete this file & create another PR for the input docs. thanks